### PR TITLE
DTSPO- 2879 - Renaming charts

### PR DESCRIPTION
**Pull request by endakelly**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DTSPO-2879

### Change description ###
Renaming crumble to apple
Renaming plum to toffee

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

File: charts/apple-recipe-backend/Chart.yaml [hunk 1]
```diff
@@ -1,11 +1,10 @@
-name: crumble-recipe-backend
+name: apple-recipe-backend
 apiVersion: v2
 home: https://github.com/hmcts/cnp-plum-recipes-service
```
Comment (mokainos) [inline charts/apple-recipe-backend/Chart.yaml:3]: This needs updating to sds-toffee-recipes-service

File: charts/toffee-recipe-backend/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+name: toffee-recipe-backend
+apiVersion: v2
+home: https://github.com/hmcts/cnp-toffee-recipes-service
```
Comment (mokainos) [inline charts/toffee-recipe-backend/Chart.yaml:3]: https://github.com/hmcts/sds-toffee-recipes-service

File: charts/apple-recipe-backend/Chart.yaml [hunk 3]
```diff
@@ -1,11 +1,10 @@
...
 version: 0.2.2
-description: This is for Jenkins pipeline tests of plum /crumble app.
+description: This is for Jenkins pipeline tests of plum /apple app.
```
Comment (mokainos) [inline charts/apple-recipe-backend/Chart.yaml:5]: toffee/apple

File: charts/apple-recipe-backend/values.sandbox.template.yaml [hunk 4]
```diff
@@ -2,6 +2,6 @@ java:
   # Don't modify below here
   image: ${IMAGE_NAME}
   ingressHost: ${SERVICE_FQDN}
-  aadIdentityName: crumble
+  aadIdentityName: apple
```
Comment (timja) [inline charts/apple-recipe-backend/values.sandbox.template.yaml:5]: this needs to be added into flux,
apple-sbox-mi

cft equivalent here: https://github.com/hmcts/cnp-flux-config/blob/master/apps/cnp/sbox/base/crumble-identity.yaml

**Review by timja — CHANGES_REQUESTED** [review]

**Review by mokainos — CHANGES_REQUESTED** [review]

File: charts/apple-recipe-backend/values.sandbox.template.yaml (hunk 4 above)
Comment (endakelly) [reply charts/apple-recipe-backend/values.sandbox.template.yaml:5]: https://github.com/hmcts/shared-services-flux/pull/1037

**Review by timja — APPROVED** [review]

**Pull request merged**